stack.yml

```
@@ -7,10 +7,12 @@ secrets:
     external: true
   opam-repo-ci-oauth:
     external: true
+  ocaml-ci-webhook-secret:
```

```suggestion
  opam-repo-ci-webhook-secret:
```

I'm using a rough naming convention of "<service>-<application-cli-name>"  to avoid clashes when we run multiple instances on the same docker swarm.